src/components/ContactForm.astro

```diff
@@ -0,0 +1,78 @@
+---
+import cover from "@/assets/images/backgrounds/KMUTT_LX_Building.png";
+import Button from "./Button.astro";
+---
+
+<div class="p-2">
+  <div class="relative py-20 rounded-2xl md:rounded-3xl overflow-clip">
+    <img
+      src={cover.src}
+      class="absolute top-0 left-0 w-full h-full object-cover"
+    />
+    <div class="absolute bg-black opacity-70 w-full h-full top-0"></div>
+    <div
+      class="relative  justify-around mx-auto px-12 space-y-4 md:space-y-0 md:flex md:flex-row flex flex-col"
+    >
+      <div class="text-white w-full">
+
+        <h1 class="text-red-600 font-bold">Contact</h1>
+        <h1 class="mt-3 md:mt-6 text-3xl md:text-4xl font-semibold">Connect with us</h1>
+        
+
+          <p class="text-sm font-thin md:text-lg mt-4 w-full md:w-3/4">React out to learn about upcoming events, speaking opportunities, or how to get involved in spreading innovative ideas at KMUTT.</p>
```

Comment:
Typo

Reach out to learn about upcoming events, speaking opportunities, or how to get involved in spreading innovative ideas at KMUTT